scripts/docker/software_router/config.groovy

```diff
@@ -0,0 +1,14 @@
+rabbitmq.host = "rabbitmq"
```

Comment:
100% sure I will forget to update this file. Why making a copy of the file which is in the repository root ? 
Much more flexible and less error prone to copy the file in the Docker build context in the CI script.